### DISCUSSION TITLE
How to repackage this folder decompressed?

### DISCUSSION THREAD

**Discussion by Leet-Lyn** [gildas-lormeau/SingleFileZ#171] [Q&A] [answered]
What an extraordinary addon.

I believe that Html generated by SingleFile(Z) is actually zips that can be decompressed by Winzip or 7zip. I want to compress the images inside by myself.

So I'd like to find out that. How to repackage this folder so that I can read it with a browser again ?

Thank you very much.

**Comment by gildas-lormeau** [comment, marked as answer]
The simplest way is to extract the file, make your changes, run a (simple) HTTP server from the folder where you extracted the file, and save the page with SingleFile(Z) again via http://localhost.

**Reply by Leet-Lyn** [reply]
OK. I know it，thanks.
